### DISCUSSION TITLE
Android UX Floor Scan Prompts and Close Button

### DISCUSSION THREAD

**Discussion by allenwixted** [google/model-viewer#2014] [Q&A] [answered]
Hi there, 

We are using Model Viewer on our viewer https://www.get-plop.com/viewer?product_id=355 but I'm struggling to see why we don't get the gif of the hand and phone prompting the user to scan their floor while viewing in Android. This is prompting me to question if we are missing any other UX features provided by model viewer or scene viewer? 

Secondly, the closing X button appears in different spots (top left on pixel 3A) one of which is top tight, behind the camera cutout of the Samsung s10. Where can this placement be set? 

Many thanks in advance, 
Allen

**Comment by elalish** [comment, marked as answer]
What you're experiencing is the difference between the `webxr` and `scene-viewer` AR modes. The first is in the browser while the second is a separate native app, and there a different pros and cons between them. You can choose to only allow one or the other if you want. If you choose `webxr` mode, all this UX is customizable through DOM slots: https://modelviewer.dev/examples/augmentedreality/#webXR including the exit button, which we moved by default to get away from the camera cutout. We have very little customization control over scene-viewer.

**Reply by allenwixted** [reply]
Many thanks Emmett for the speed replies! Roger that, will get right on it - thanks so much
